### PR TITLE
high: cibconfig: Use correct CIB as patch base (bsc#1127716)

### PR DESCRIPTION
**Pull request by krig**
crmsh was passing a sanitized CIB version as the basis for
patches. This works as long as the CIB is already sanitized
(apparently almost always the case) but generates strange
diffs otherwise.

### CODE REVIEW

File: crmsh/xmlutil.py [hunk 1]
```diff
@@ -675,6 +684,22 @@ def sanitize_cib(doc):
...
+    """
+    Custom version of sanitize_cib which
+    doesn't sort container children, to use
+    for processing the original CIB when
+    generating a patch to apply using crm_diff.
+    """
+    xml_processnodes(doc, is_status_node, rmnodes)
+    # xml_processnodes(doc, true, printid)
+    # xml_processnodes(doc, is_emptynvpairs, rmnodes)
+    # xml_processnodes(doc, is_emptyops, rmnodes)
```
Comment (diegoakechi) [inline crmsh/xmlutil.py]: I would maybe remove the commented lines to make it easier to read and visualize the difference between the two functions.

Comment (krig) [reply crmsh/xmlutil.py]: alright, makes sense, I'll do that.

**Comment by diegoakechi** [comment]
Wow! Nice and very complex new test case...  Well done @krig

**Comment by liangxin1300** [comment]
Great Job!

**Pull request merged**